src/test/java/uk/gov/hmcts/sscs/schema/RoboticsSchemaTest.java

```
@@ -0,0 +1,153 @@
+        schema.validate(jsonData);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void givenInvalidInputForHearingLoop_throwExceptionWhenValidatingAgainstSchema() throws ValidationException {
+        jsonData =  updateEmbeddedJson(jsonData.toString(), "Bla", "hearingArrangements", "hearingLoop");
+        schema.validate(jsonData);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void givenInvalidInputForAccessibleHearingRoom_throwExceptionWhenValidatingAgainstSchema() throws ValidationException {
+        jsonData =  updateEmbeddedJson(jsonData.toString(), "Bla", "hearingArrangements", "accessibleHearingRoom");
+        schema.validate(jsonData);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void givenInvalidInputForDisabilityAccess_throwExceptionWhenValidatingAgainstSchema() throws ValidationException {
+        jsonData =  updateEmbeddedJson(jsonData.toString(), "Bla", "hearingArrangements", "disabilityAccess");
+        schema.validate(jsonData);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void givenInvalidInputForHearingDatesCantAttend_throwExceptionWhenValidatingAgainstSchema() throws ValidationException {
+        jsonData = new JSONObject(jsonData.toString().replace("2018-08-12", "2018/08/12"));
+        schema.validate(jsonData);
+    }
+
+    private static JSONObject updateEmbeddedJson(String json, String value, String... keys) {
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        JSONObject jsonObject = null;
+        try {
+            Map map = objectMapper.readValue(json, Map.class);
+
+            Map t = map;
+            for (int i = 0; i < keys.length - 1; i++) {
+                t = (Map) t.get(keys[i]);
+            }
+
+            t.put(keys[keys.length - 1], value);
+
+            jsonObject = new JSONObject(objectMapper.writeValueAsString(map));
+        } catch (IOException e) {
+            e.printStackTrace();
```

It will be better to throw the exception.